swaglyrics/__main__.py

```diff
@@ -25,10 +25,15 @@ def main():
 	print("Updated unsupported.txt successfully.")
 
 	parser = argparse.ArgumentParser(
+		prog="Swaglyrics",
+		usage='%(prog)s [options]',
 		description="Get lyrics for the currently playing song on Spotify. Either --tab or --cli is required.")
 
-	parser.add_argument('-t', '--tab', action='store_true', help='Display lyrics in a browser tab.')
-	parser.add_argument('-c', '--cli', action='store_true', help='Display lyrics in the command-line.')
+	group = parser.add_mutually_exclusive_group()
+	group.add_argument('-t', '--tab', action='store_true', help='Display lyrics in a browser tab.')
```

Comment:
How's this implementation better? You should make sure to explain why you change something instead of magically assuming that I know what you're thinking 😉

Reply:
Haha Okay sorry, if you'll give the command like this `swaglyrics -t -c` then it'll not give an error instead it'll open up the browser, now with the help of this `mutually_exclusive_group`, only one of the commands can be selected out of the group, if both of them are selected then it'll not run anything instead will give an error like this - `Swaglyrics: error: argument -t/--tab: not allowed with argument -c/--cli`

Reply:
Sounds good 👍